lib/SimpleRadioButton.js

```diff
@@ -99,6 +105,25 @@ var RadioButton = React.createClass({
   },
   render: function () {
     var c = this.props.children
+
+    var idSeparator = (this.props.idSeparator) ? this.props.idSeparator : '|'
+    var idSeparatorAccessibilityLabelIndex = (this.props.accessibilityLabel)
+    ? this.props.accessibilityLabel.indexOf(idSeparator) : -1
+    var idSeparatorTestIdIndex = (this.props.testID)
+    ? this.props.testID.indexOf(idSeparator) : -1
+
+    var accessibilityLabel = (this.props.accessibilityLabel)
+    ? (idSeparatorAccessibilityLabelIndex !== -1
+    ? this.props.accessibilityLabel.substring(0, idSeparatorAccessibilityLabelIndex) : this.props.accessibilityLabel) : 'radioButton'
+    var testID = (this.props.testID)
+    ? (idSeparatorTestIdIndex !== -1
+    ? this.props.testID.substring(0, idSeparatorTestIdIndex) : this.props.testID) : 'radioButton'
+
+    var accessibilityLabelIndex = (this.props.accessibilityLabel && idSeparatorAccessibilityLabelIndex !== -1)
+    ? this.props.accessibilityLabel.substring(idSeparatorAccessibilityLabelIndex + 1) : ''
+    var testIDIndex = (this.props.testID && testIDIndex !== -1)
```

Comment:
This doesn't work - should be:
`    var testIDIndex = (this.props.testID && idSeparatorTestIdIndex !== -1)
      ? this.props.testID.substring(idSeparatorTestIdIndex + 1) : '';`